README.md

```diff
@@ -133,6 +133,20 @@ context, the Docker object is removed automatically, even if an exception occurs
 * All Docker objects and the Docker client are safe to use with multithreading and multiprocessing.
 * Display the commands called and the environment variables used by setting the environment variable `PYTHON_ON_WHALES_DEBUG=1`.
 
+### STDOUT and STDERR handling
+
+_While this is by no means an exhaustive explanation it conveys the overall philosophy of python-on-whales_
+
+Most python-on-whales methods capture `STDOUT` and `STDERR`.
+
+`STDERR` is only used to raise appropriate exceptions when the underlying docker process fails.
+
+Many of these methods, like `docker.volume.prune`, return `STDOUT` in its raw form. You can ignore the `STDOUT` or perhaps log it.
```

Comment:
That's not really true. I don't know if it was the purpose of your last PR, but stdout is not returned. If it were to be returned (which is a valid thing to do), we would parse stdout and output a list of volume IDs (strings).

In you last PR, there was `run(full_cmd)`, to return stdout, we would need to do `return run(full_cmd)`, but I believe the best thing to do would be 
```python
output = run(full_cmd)
if output == "":
    return []
else:
    return output.splitlines()
```

Reply:
Sorry. I missed something. `utils.run` returns stdout but `docker.volume.prune` does not return `utils.run`.

Agreed it could be nice to do the above, which is what I thought happened when I wrote this addition to the README.  It seems a little involved but entirely doable. 

Looks like this PR will get a lot bigger, and probably not be done for a bit. :)

(Also, I wrote my previous PR because `volume.prune`'s output was appearing in my program's stdout while no other prune method's was.)

Reply:
No worries. I'm not sure many people use the output of docker.volume.prune. So don't worry about it. You can just remove the section concerning docker.volume.prune in this PR and then we can merge it :)